Pylint `possibly-used-before-assignment` detections in `copy_fix.py`

Seems like pylint got updated so the style check fails on master currently.

```
pylint 3.2.0
astroid 3.2.0
Python 3.8.10 (default, Nov 22 2023, 10:22:35) 
[GCC 9.4.0]
************* Module copy_fix
scripts/languages/copy_fix.py:16:35: E0606: Possibly using variable 'local' before assignment (possibly-used-before-assignment)
scripts/languages/copy_fix.py:20:4: E0606: Possibly using variable 'supported' before assignment (possibly-used-before-assignment)

-----------------------------------
Your code has been rated at 9.97/10
```

--- a/scripts/languages/copy_fix.py
+++ b/scripts/languages/copy_fix.py
@@ -8,10 +8,8 @@ def copy_fix(infile, delete_unused, append_missing, delete_empty):
 	with open(infile, encoding="utf-8") as f:
 		content = f.readlines()
 	trans = twlang.translations(infile)
-	if delete_unused or append_missing:
-		local = twlang.localizes()
-	if append_missing:
-		supported = []
+	local = twlang.localizes()
+	supported = []
 	for tran, (start, expr, end) in trans.items():
 		if delete_unused and tran not in local:
 			content[start:end] = [None]*(end-start)

